x-pack/plugins/security_solution/cypress/e2e/detection_alerts/enrichments.cy.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getNewRule } from '../../objects/rule';
+import {
+  NUMBER_OF_ALERTS,
+  HOST_RISK_HEADER_COLIMN,
+  USER_RISK_HEADER_COLIMN,
+  HOST_RISK_COLUMN,
+  USER_RISK_COLUMN,
+  ACTION_COLUMN,
+  CLOSE_FLYOUT,
+} from '../../screens/alerts';
+import { ENRICHED_DATA_ROW } from '../../screens/alerts_details';
+import { esArchiverLoad, esArchiverUnload } from '../../tasks/es_archiver';
+
+import { createCustomRuleEnabled } from '../../tasks/api_calls/rules';
+import { cleanKibana, deleteAlertsAndRules } from '../../tasks/common';
+import { waitForAlertsToPopulate } from '../../tasks/create_new_rule';
+import { expandFirstAlert, scrollAlertTableColumnIntoView } from '../../tasks/alerts';
+
+import { login, visit } from '../../tasks/login';
+
+import { ALERTS_URL } from '../../urls/navigation';
+
+describe('Enrichment', () => {
+  before(() => {
+    cleanKibana();
+    esArchiverLoad('risky_hosts');
+    esArchiverLoad('risky_users');
+    login();
+  });
+
+  after(() => {
+    esArchiverUnload('risky_hosts');
+    esArchiverUnload('risky_users');
+  });
```

Comment:
Please, unload the `risky_hosts_updated` archive as well after the test execution.